### PR TITLE
Add checkpoint files for all pre-trained models

### PR DESCRIPTION
**Pull request by roshni-b**
Updated [pre-trained_models.md](https://github.com/esowc/wildfire-forecasting/blob/pretrained/src/model/checkpoints/pre_trained/pre-trained_models.md) with links to the .ckpt files and associated metadata (Inputs, Months, Pre-processing, Epochs) for all trainings. The checkpoints files are saved in [2_1](https://github.com/esowc/wildfire-forecasting/tree/pretrained/src/model/checkpoints/pre_trained/2_1), [4_10](https://github.com/esowc/wildfire-forecasting/tree/pretrained/src/model/checkpoints/pre_trained/4_10), [4_14](https://github.com/esowc/wildfire-forecasting/tree/pretrained/src/model/checkpoints/pre_trained/4_14) & [7_21](https://github.com/esowc/wildfire-forecasting/tree/pretrained/src/model/checkpoints/pre_trained/7_21) folders.

Another PR with Accuracy and Error plots for these runs to follow.

### CODE REVIEW

**Review by lazyoracle — COMMENTED** [review]
All good!
